README.md

```diff
@@ -44,4 +44,39 @@ pip install zep-python
 
 ```bash
 poetry add zep-python
-```
\ No newline at end of file
+```
+
+## Zep cloud Installation
```

Comment:
Please mention the required environment variables in the README, including the Collection name.